### ISSUE TITLE
Fix globals member expression bug

### ISSUE THREAD

**Issue by tmpfs** [LavaMoat/basalt#18]
When generating a policy for the `browserslist` package, `indexOf` is incorrectly being detected as global.

This is likely another flaw with the member expression handling.

**Comment by tmpfs** [comment]
This is the problematic expression:

```javascript
(versionA + '.').indexOf(versionB + '.')
```

Care should be taken a fix does not break this expression which should yield `process.platform` and `process.env.TERM`:

```javascript
const isSimpleWindowsTerm = process.platform === 'win32' && !(process.env.TERM || '').toLowerCase().startsWith('xterm');
```

**Comment by tmpfs** [comment]
Fixed in e9f28c6219b52ba66d6e173fe4d3941b1be0adfe.
